lcls-twincat-common-components/Library/Devices/PPM/FB_PPM_PowerMeter.TcPOU

```diff
@@ -6,6 +6,9 @@ VAR_INPUT
     fTempSP: REAL;
     bVetoTempFFO: BOOL;
     sDeviceName: STRING;
+    eEnumGet: E_PPM_States;
+    stYAxisState: DWORD;
```

Comment:
Review note to help myself: axis state dword is implicitly an enum as described on this page: https://infosys.beckhoff.com/english.php?content=../content/1033/tcplclib_tc2_mc2/70133899.html&id=